src/lib/api/api.ts

```diff
@@ -0,0 +1,70 @@
+import { fetchPokemonData, fetchSpeciesData, fetchEvolutionData, fetchPokemonListData } from './fetch';
+import { GetPokemonParams, PokemonInfo, Language } from './type';
+import axiosInstance from './instance';
+import {
+  getPokemonName,
+  getPokemonGenus,
+  getFlavorText,
+  getPokemonTypes,
+  getImages,
+  getAbilities,
+} from './getPokemonData';
+
+// 포켓몬 한마리의 데이터
+export const getPokemonInfo = async ({ number, language }: GetPokemonParams) => {
+  try {
+    const pokemonData = await fetchPokemonData(number);
+    const speciesData = await fetchSpeciesData(number);
+
+    const { id, weight, height, abilities, types } = pokemonData;
+
+    const name = getPokemonName(speciesData, language);
+    const genus = getPokemonGenus(speciesData, language);
+    const flavor = getFlavorText(speciesData, language);
+    const typeList = await getPokemonTypes(types, axiosInstance, language);
+    const { pokemonImage, pokemonShinyImage } = getImages(pokemonData);
+    const abilityList = await getAbilities(abilities, axiosInstance, language);
+
+    return {
+      id,
+      weight,
+      height,
+      name,
+      genus,
+      flavor,
+      typeList,
+      image: pokemonImage,
+      shiny: pokemonShinyImage,
+      abilityList,
+    } as PokemonInfo;
+  } catch (error) {
+    console.error(error);
+  }
+};
+
+// 포켓몬 도감 리스트 데이터
+// Todo 더보기 버튼 클릭시 offset와 limit로 추가 데이터 불러오도록 하기
+export const getPokemonAllList = async () => {
+  // offset: 몇번째 부터 불러올지 정하는 값, limit: 몇개의 데이터를 불러올지 정하는 값
+  const pokemonAllListResponse = await fetchPokemonListData({ offset: 0, limit: 20 });
```

Comment:
인자가 전달이 안되고 있네용 이렇게 변경되면 좋을 것 같습니다
  ```suggestion
export const getPokemonAllList = async (offset: number = 0, limit: number = 20) => {
  // offset: 몇번째 부터 불러올지 정하는 값, limit: 몇개의 데이터를 불러올지 정하는 값
  const pokemonAllListResponse = await fetchPokemonListData({ offset, limit });
  ```

Reply:
넵 해당값 인자로 전달될 수 있도록 수정했습니다!